sdk/python/tests/integration/feature_repos/universal/data_sources/file.py

```diff
@@ -35,7 +35,7 @@ def create_data_sources(
             event_timestamp_column=event_timestamp_column,
             created_timestamp_column=created_timestamp_column,
             date_partition_column="",
-            field_mapping=field_mapping or {"ts_1": "ts", "id": "driver_id"},
```

Comment:
intentional removal?

Reply:
Yes, intentional to be consistent with the other sources